client/src/components/Common/CouponCode/index.js

```diff
@@ -0,0 +1,213 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { Input } from 'antd';
+
+import { API_COUPON_URL } from '../../../constants/apiRoutes';
+
+import { createStartEndDate, getDiscountDays } from '../../../helpers';
+
+// Typography
+import * as T from '../Typography';
+
+import { colors } from '../../../theme';
+
+// checks db with code
+const makeRequest = async _code => {
+  try {
+    const {
+      data: {
+        data: [couponInfo],
+      },
+    } = await axios.get(`${API_COUPON_URL}?code=${_code}`);
```

Comment:
just to let you know that axios accepts params now
```suggestion
    } = await axios.get(`${API_COUPON_URL}`, { params: { code: _code } });
```